components/bio/bio.scss

```diff
@@ -24,22 +24,13 @@
   }
 
   .name {
-    font-size: 1.75rem;
-    font-weight: 500;
-  }
-
-  .other-meta {
-    font-size: 0.75rem;
-    font-weight: 600;
+    font-size: 1.375rem;
```

Comment:
We don't need any custom SCSS as `name` should be in `h5-heading` style.

Also, other than nav links (as the nav link style hasn't been finalized yet), elements on the site shouldn't have custom font size or line height. If you spot anything from the mockups that has custom font style you should flag it to the designer.

Reply:
Okay, this means that the profile name that's wrapped in an `<h1>` will have the `.h5-heading` class. If that's 🆒 then 👍

Reply:
This is exactly why we have these classes - so that we can design the elements the way we want while making their markup semantic

Reply:
Perfecttttt